src/obb_rules/game_progress.cljc

```diff
@@ -0,0 +1,54 @@
+(ns obb-rules.game-progress
```

Comment:
Missing a space on the require.

Also, could you change the ns of the files you created to have the following format:

```clojure
(ns ^{:added "3.0.0"
      :author "Joaquim Torres"}
  ns-name
  "ns desc"
...
```

I started doing this at some point, inspired by clojure repo. I feel ti brings a nice sense of timeline.